src/Tribe/Importer/Options.php

```diff
@@ -124,4 +125,30 @@ public static function get_possible_stati() {
 
 		return apply_filters( 'tribe_import_possible_stati', $stati );
 	}
+	public static function get_default_encoding_status( $type = 'csv' ) {
```

Comment:
Add a line break between the previous function and this one. Also add a docblock for this method, please.